components/AccountsFooter.vue

```diff
@@ -1,54 +1,72 @@
 <template>
```

Comment:
It's easier to review this diff if you go to the ⚙️ icon above and hide whitespace changes.